sources/packages/forms/src/form-definitions/studentdisabilityappeal.json

```diff
@@ -0,0 +1,369 @@
+        "style": "",
+        "page": "",
+        "left": "",
+        "top": "",
+        "width": "",
+        "height": ""
+      },
+      "type": "panel",
+      "label": "Panel",
+      "breadcrumb": "default",
+      "tabindex": "",
+      "input": false,
+      "tableView": false,
+      "components": [
+        {
+          "autofocus": false,
+          "input": true,
+          "tableView": true,
+          "inputType": "radio",
+          "label": "Are you a student with disabilities and wish to be assessed for disability funding?",
+          "key": "studentNewPDPPDStatus",
+          "values": [
+            {
+              "value": "yes",
+              "label": "Yes, I have disability and want this application to be assessed for disability funding.",
+              "shortcut": ""
+            },
+            {
+              "value": "noIWantToAccessOtherFundingTypes",
+              "label": "No, I want to access other funding types while I wait for a verification decision.",
+              "shortcut": ""
+            },
+            {
+              "value": "noIDoNotHaveADisability",
+              "label": "No, I do not have a disability.",
+              "shortcut": ""
+            }
+          ],
+          "defaultValue": "",
+          "protected": false,
+          "fieldSet": false,
+          "persistent": true,
+          "hidden": false,
+          "clearOnHide": true,
+          "validate": {
+            "required": true,
+            "custom": "",
+            "customPrivate": false
+          },
+          "type": "radio",
+          "labelPosition": "top",
+          "optionsLabelPosition": "right",
+          "tags": [],
+          "conditional": {
+            "show": "",
+            "when": null,
+            "eq": ""
+          },
+          "properties": {},
+          "lockKey": true
+        }
+      ],
+      "placeholder": "",
+      "prefix": "",
+      "suffix": "",
+      "multiple": false,
+      "defaultValue": null,
+      "protected": false,
+      "unique": false,
+      "persistent": false,
+      "clearOnHide": false,
+      "refreshOn": "",
+      "redrawOn": "",
+      "dataGridLabel": false,
+      "labelPosition": "top",
+      "description": "",
+      "errorLabel": "",
+      "autofocus": false,
+      "dbIndex": false,
+      "customDefaultValue": "",
+      "calculateValue": "",
+      "calculateServer": false,
+      "widget": null,
+      "validateOn": "change",
+      "validate": {
+        "required": false,
+        "custom": "",
+        "customPrivate": false,
+        "strictDateValidation": false,
+        "multiple": false,
+        "unique": false
+      },
+      "allowCalculateOverride": false,
+      "encrypted": false,
+      "showCharCount": false,
+      "showWordCount": false,
+      "allowMultipleMasks": false,
+      "addons": [],
+      "tree": false,
+      "lazyLoad": false,
+      "id": "efu9xee"
+    },
+    {
+      "label": "applicationId",
```

Comment:
Did you manually add the applicationId and applicationStatus? If so can you please double check if that is sent to the form data and being used? Because I doubt if it is used.  And these values were automatically added when we were using formflow.ai. 

@guru-aot  do you remember this?

Reply:
No, I just copied from the others that have these hidden fields but we are actually not using that.

Reply:
I removed the hidden fields and tested. Thanks!

Reply:
Yes its automatically generated. WE can remove it.